tools/mesh_sampling.cpp

```diff
@@ -135,7 +169,8 @@ uniform_sampling (vtkSmartPointer<vtkPolyData> polydata, size_t n_samples, bool
   {
     Eigen::Vector4f p;
     Eigen::Vector3f n;
-    randPSurface (polydata, &cumulativeAreas, totalArea, p, calc_normal, n);
+    Eigen::Vector3i c;
```

Comment:
This can as well be `Eigen::Vector4f c`, which can be passed directly to the existing version of `randomPointTriangle()`.